packages/@n8n/chat/src/components/Input.vue

```diff
@@ -163,7 +162,7 @@ function onKeyDown(event: KeyboardEvent) {
 					v-if="isFileUploadAllowed"
 					:disabled="isFileUploadDisabled"
 					class="chat-input-send-button"
-					@click="() => openFileDialog()"
+					@click="() => openFileDialog({ accept: unref(allowedFileTypes) })"
```

Comment:
Can you please extract this to an `@click="onOpenFileDialog"` ? Otherwise the function gets recreated on every render.